features/support/0001_world.rb

```diff
@@ -4,7 +4,6 @@
 World(PlanHelpers)
 World(DummyAttachments)
 
-World(JsHelpers)
```

Comment:
What's this file for?

Reply:
Looks like it just loads all the helpers. I was tempted to add this `World(Whatever)` in the same `whatever.rb` file, like `World(CapybaraExtensions)`

But then I thought - probably the order is important? :thinking: 
by the name of the file (starting with `0`), I guess it was done so to ensure the order of loading those helpers, although I am not sure whether it is needed. I decided not to change it here.

Reply:
@akostadinov I guess both. Anyway it's clear now.